backend/src/services/telemetry/telemetry-types.ts

```diff
@@ -8,7 +8,11 @@ export enum PostHogEventTypes {
   UserSignedUp = "User Signed Up",
   SecretRotated = "secrets rotated",
   SecretScannerFull = "historical cloud secret scan",
-  SecretScannerPush = "cloud secret scan"
+  SecretScannerPush = "cloud secret scan",
+  ProjectCreated = "project created",
+  IntegrationCreated = "integration created",
+  MachineIdentityCreated = "machine identity created",
+  UserOrgInvitation = "user org invitation"
```

Comment:
Could you please capitalize the new events? Don't change the old ones, but i think it would be good to have that as the naming convention for the new ones